critical-css.js

```diff
@@ -35,5 +37,21 @@ function findCriticalCss(cssString, url) {
     timeout: 120000,
     width: 4096,
     height: 2160,
-    blockJSRequests: false
+    blockJSRequests: false,
+    screenshots: {
+      basePath: getScreenshotPath(url),
+      type: 'jpeg',
+      quality: 100
+    }
   })}
+
+// Returns a relative path for the screenshot using the url
+function getScreenshotPath(url) {
+  const screenshotsDir = "critical-css-screenshots";
+  // sanitize the url to use it as a filename
+  const filename = url.replace(/[?/:<>"|*]/g, "-");
```

Comment:
Nice! I'm assuming we tested this for all of our paths, right?

Reply:
Yes! They end up something like this:
```
https---www.ifixit.com-Answers-View-473839-iOS+devices+connecting+and+disconnecting+rapidly.-before.jpg
```

Reply:
Sweet 👍